antiplag/models.py

```diff
@@ -65,8 +66,9 @@ def process_file(self):
         return extract_text_from_file(self.file.path)
 
     def process_raw_text(self):
-        # Replace with actual process raw text method.
-        return 'text'
+        os.environ["w2n.lang"] = self.language
```

Comment:
What if there are more parallel text_processing functions running at the same time? Won't the latter overwrite the environment variable? Cannot we solve this using parameters?

Reply:
Agreed, setting env variable does not seem as best approach.

Reply:
downloaded module word2numberi18n use that by default. there are other options but none of them is great for us.
this is searching process of that module for language definition:
![image](https://user-images.githubusercontent.com/29302158/102016655-4c3baa00-3d62-11eb-8694-2dda944de58d.png)

Reply:
This is so wrong - why doesn't it accept lang as a parameter?

Cannot we find any better alternative?

Reply:
This is only module i found that works for any language

Reply:
maybe i can modify this module. Add some function for setting language before call..?

Reply:
i can change this function 
![image](https://user-images.githubusercontent.com/29302158/102016934-d9cbc980-3d63-11eb-8d05-1bd411e90b23.png)
like this and add line
![image](https://user-images.githubusercontent.com/29302158/102016946-f6680180-3d63-11eb-9918-4d5ab74d26b4.png)

Reply:
Well, not sure but dependencies should not be changed - only if we forked it and created our own package.

Reply:
okay so what we do with this?
there are no other modules that can work with slovak language

Reply:
Alright, for now, leave it like it is , just please add a commentary
`#TODO: Would not work in parallel`


In the close future: 

@OliverKanat What do you think of creating our own fork and package and refactoring the function to accept the language argument before checking all the other possibilities (env, sys lang etc.)

Or @Scr0ll , seems like the package is not dead and creating an issue with PR there would be beneficial.
https://github.com/bastie/w2ni18n

Reply:
I think even Java is better solution than this.

Reply:
I think creating an issue with PR is good idea and we can skip processing words to numbers until it's resolved.